### PR TITLE
option to display mesh with color

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description

option to display mesh with color
# Checklist

- [ ] Doxygen documentation of the code completed (classes, methods, types, members...)
- [ ] Check if it follows the tools structure described in [CONTRIBUTING.md](https://github.com/DGtal-team/DGtalTools-contrib/blob/master/CONTRIBUTING.md)
- [ ] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtalTools-contrib/blob/master/ChangeLog.md) added.
- [ ] Update the readme with potentially a screenshot of the tools if it applies. 
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).

### CODE REVIEW

File: visualisation/graphViewer.cpp [hunk 1]
```diff
@@ -54,6 +55,9 @@ int main( int argc, char** argv )
     ("inputEdge,e", po::value<std::string>(), "input file containing the edge list.")
     ("inputRadii,r", po::value<std::string>(), "input file containing the radius for each vertex.")
     ("ballRadius,b", po::value<double>()->default_value(1.0), "radius of vertex balls.")
+    ("addMesh,m", po::value<std::string>(), "add mesh in the display.")
```
Comment (akrah) [inline visualisation/graphViewer.cpp]: ou "inputMesh" pour être cohérent avec les autres options ?

File: visualisation/graphViewer.cpp [hunk 2]
```diff
@@ -54,6 +55,9 @@ int main( int argc, char** argv )
...
+    ("colorMesh", po::value<std::vector<unsigned int> >()->multitoken(), "specify the color mesh.")
+    ("colorEdges", po::value<std::vector<unsigned int> >()->multitoken(), "specify the color of edges.")
```
Comment (akrah) [inline visualisation/graphViewer.cpp]: meshColor and edgeColor ?

**Comment by akrah** [comment]
After option names, all is good for me !

File: visualisation/graphViewer.cpp (hunk 1 above)
Comment (kerautret) [inline visualisation/graphViewer.cpp]: yes but it is not really directly related to graph for me

File: visualisation/graphViewer.cpp (hunk 2 above)
Comment (kerautret) [inline visualisation/graphViewer.cpp]: yes

**Comment by akrah** [comment]
All is ok for me, you can merge !

File: visualisation/graphViewer.cpp (hunk 1 above)
Comment (akrah) [inline visualisation/graphViewer.cpp]: Yes, ok for that !

**Comment by kerautret** [comment]
thanks merging

**Pull request merged**